### DISCUSSION TITLE
Music Library Support

### DISCUSSION THREAD

**Discussion by BitlessByte0** [PlexRipper/PlexRipper#358] [Ideas]
Hello, I'm wondering what is the status of being able to download from music libraries. These are commonly used for audiobooks which seem (at least to me) to be a harder to find commodity in the scene. Being able to download a friend's file that is now _dead_ elsewhere would be very useful!

**Comment by JasonLandbridge** [comment]
Hi, 

It's quite low on the list due to other more in demand features. You're honestly the first one who asks about it :p

See the very rough plan here: https://github.com/orgs/PlexRipper/projects/2?query=sort%3Aupdated-desc+is%3Aopen

But if it something more people want then I'm happy to work on it
